authentication/authentication-styles.css

```diff
@@ -0,0 +1,76 @@
+body {
+    margin-left: 20px;
+    margin-right: 20px;
+    margin-top: 100px;
+    margin-bottom: 50px;
+}
+
+.card {
+    max-width: 50vw;
+    min-width: fit-content;
+    height: 50vh;
```

Comment:
- Set the height fixed here will make the text overflow. Unset this would work properly
<img width="770" alt="Screenshot 2023-10-31 at 11 15 35" src="https://github.com/comtam-mohanh/frontend-1/assets/55119976/dfbe7776-954a-434a-b17f-d6167295912a">